vault-minikube/step7.md

```diff
@@ -0,0 +1,39 @@
+Deploy the webapp in Kubernetes by applying the file `deployment-01-webapp.yml`{{open}}.
+
+```shell
+kubectl apply --filename deployment-01-webapp.yml
+```{{execute T1}}
+
+The web application runs as a pod within the `default` namespace.
+
+Get all the pods within the `default` namespace.
+
+```shell
+kubectl get pods
+```{{execute T1}}
+
+The web application pod is displayed here as the pod prefixed with `webapp`.
+
```

Comment:
Perhaps, you want to add a note saying something like...

"Wait until the `webapp` status becomes **Running** and Ready (`1/1`)."

For those inpatient students. ;)

Reply:
I like this change.